query/transformation.go

```diff
@@ -42,7 +42,7 @@ func transformTimeseries(series api.Timeseries, transform transform, parameters
 // timerangeScale takes an api.Timerange and computes the "scale" or duration of one sample (in seconds).
 // It is useful for transformations that normalize on time (like derivative or integral).
 func timerangeScale(timerange api.Timerange) float64 {
-	return float64(timerange.Resolution)
+	return float64(timerange.Resolution())
```

Comment:
Derivative and such will still work if you keep all units as millis instead of seconds. If the final result needs to be in seconds, that's fine, but we should only do the conversion before sending back the result.